app/assets/stylesheets/semantic-ui/modules/_search.scss

```diff
@@ -172,9 +172,6 @@
 ---------------------*/
 
 .ui.search > .prompt:focus {
-  border-color: rgba(34, 36, 38, 0.35);
-  background: #FFFFFF;
-  color: rgba(0, 0, 0, 0.95);
```

Comment:
I really don't like not having a styling for focused state. https://www.viget.com/articles/managing-focus-styles-without-breaking-accessibility